### ISSUE TITLE
Tutorial idea : How do I know it's working

### ISSUE THREAD

**Issue by ebrent8** [MinnowBoard-org/website#281]
For the list…perhaps a “How do I know its working” tutorial?...maybe a debug flow chart?

This is the official warranty statement from the /get-help page….work with the place you bought it at.
 
Seems like we need a “How do I know it works” debug tutorial…
 
 https://minnowboard.org/help/warranty

**Comment by ebrent8** [comment]
Who should do this one?  Still a priority?

**Comment by BrianOttaway** [comment]
Yes. Still a priority...for both Turbot and MB3. @keerock or @mark
